### PR TITLE
PRO-416 Rename xDai to Gnosis

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
Rename xdai to gnosis.

https://user-images.githubusercontent.com/78720269/186818136-393b120a-4440-45d1-b041-ac069f1b8db2.mp4

### CODE REVIEW

File: src/components/display/TokenIcon.js [hunk 1]
```diff
@@ -36,10 +36,10 @@ import type { Chain } from 'models/Chain';
 // Assets
 import IconEthereum from 'assets/icons/svg/icon-24-ethereum.svg';
 import IconBinance from 'assets/icons/svg/icon-24-binance.svg';
-import IconXdai from 'assets/icons/svg/icon-24-xdai.svg';
 import IconPolygon from 'assets/icons/svg/icon-24-polygon.svg';
 import IconAvalanche from 'assets/icons/svg/icon-24-avalanche.svg';
 import IconOptimism from 'assets/icons/svg/icon-24-optimism.svg';
+import IconGnosis from 'assets/icons/svg/icon-24-gnosis.svg';
```
Comment (IAmKio) [inline src/components/display/TokenIcon.js:42]: The native token on Gnsosis is still xDai: https://developers.gnosischain.com/for-users/getting-started-with-gc

File: src/utils/chains.js [hunk 2]
```diff
@@ -110,7 +110,7 @@ export const nativeAssetPerChain = {
     name: 'xDAI',
     symbol: XDAI,
     decimals: 18,
-    iconUrl: 'https://tokens.1inch.exchange/0x6b175474e89094c44da98b954eedeac495271d0f.png',
+    iconUrl: 'https://tokens.1inch.exchange/0x6810e776880c02933d47db1b9fc05908e5386b96.png',
```
Comment (IAmKio) [inline src/utils/chains.js:113]: Here's the icon kit for xDai: https://developers.gnosischain.com/media/media-kit#xdai-stable-coin

File: src/utils/uiConfig.js [hunk 3]
```diff
@@ -64,15 +64,16 @@ export function useChainsConfig(): { [key: Chain]: ChainConfig } {
...
+        'https://images.prismic.io/pillar-app/d8e73973-82f1-4b20-a068-fe20e21816b0_0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c_1.png?auto=compress,format',
     },
     [CHAIN.XDAI]: {
       title: t('chains.xdai'),
       titleShort: t('chainsShort.xdai'),
       iconName: 'xdai',
       color: colors.xdai,
       gasSymbol: XDAI,
-      iconUrl: 'https://tokens.1inch.exchange/0x6b175474e89094c44da98b954eedeac495271d0f.png',
+      iconUrl: 'https://tokens.1inch.exchange/0x6810e776880c02933d47db1b9fc05908e5386b96.png',
```
Comment (IAmKio) [inline src/utils/uiConfig.js:76]: As above: Here's the icon kit for xDai: https://developers.gnosischain.com/media/media-kit#xdai-stable-coin

**Review by IAmKio — CHANGES_REQUESTED** [review]
The chain name changes are good to go, however the. native token for Gnosis chain is still xDai. I've noted where these are in the PR.

File: src/utils/uiConfig.js (hunk 3 above)
Comment (hemadri-tank-bacancy) [reply src/utils/uiConfig.js:76]: I set the Gnosis icon as per zeplin ui.
image

Comment (IAmKio) [reply src/utils/uiConfig.js:76]: Whoops sorry i thought this was the native token

File: src/utils/chains.js (hunk 2 above)
Comment (hemadri-tank-bacancy) [reply src/utils/chains.js:113]: I think this icon is different from Zeplin UI. I set the Gnosis icon as per Zeplin UI.
ImageMark-LightBG

image

File: src/components/display/TokenIcon.js (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/components/display/TokenIcon.js:42]: Is this means here to show the icon as previously as it is?

Comment (hemadri-tank-bacancy) [reply src/components/display/TokenIcon.js:42]: I added an icon for the xDai native token.

**Pull request merged**